hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/StorageVolumeScannerMetrics.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.container.common.volume;
+
+import java.util.Collection;
+import org.apache.hadoop.hdds.annotation.InterfaceAudience;
+import org.apache.hadoop.metrics2.MetricsSystem;
+import org.apache.hadoop.metrics2.annotation.Metric;
+import org.apache.hadoop.metrics2.annotation.Metrics;
+import org.apache.hadoop.metrics2.lib.DefaultMetricsSystem;
+import org.apache.hadoop.metrics2.lib.MutableCounterLong;
+import org.apache.hadoop.metrics2.lib.MutableGaugeLong;
+
+/**
+ * This class captures the Storage Volume Scanner Metrics.
+ **/
+@InterfaceAudience.Private
+@Metrics(about = "Storage Volume Scanner Metrics", context = "dfs")
+public class StorageVolumeScannerMetrics {
+  public static final String SOURCE_NAME = StorageVolumeScannerMetrics.class.getSimpleName();
+
+  @Metric("number of volumes scanned in the last iteration")
+  private MutableGaugeLong numVolumesScannedInLastIteration;
+
+  @Metric("number of iterations of scanner completed since the restart")
+  private MutableCounterLong numScanIterations;
+
+  @Metric("number of data volume sets scanned")
+  private MutableGaugeLong numDataVolumesScanned;
```

Comment:
There's a mismatch in this metric's name + description and actual usage.
- `numDataVolumesScanned` describes counting the number of unique data volumes that have ever been scanned.
- `numDataVolumeScans` describes counting the number of times a scan over any data volume was run.

Currently we are using the first name with the second value. I think we can keep the usage as is and just change the name to the second one to match that. Same for the metadata volume metric below.